docs/README.md

```diff
@@ -2333,7 +2333,51 @@ after a contributor approves the PR.
 > Maintainers still have to carefully review updates to prevent exposing secrets
 > to potentially malicious updates.
 
-Here is an example workflow with a job condition to achieve this:
+#### Using the pr-review-check reusable workflow
+
+We provide a reusable workflow that automates PR review checks for Dependabot PRs and manual validation triggers. This workflow enables auto-merge on approval and validates PRs when specific labels are applied.
+
+```yaml
+name: "PR Review check"
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+  pull_request:
+    branches:
+      - develop
+    types:
+      - labeled
+
+jobs:
+  check:
+    uses: Alfresco/alfresco-build-tools/.github/workflows/pr-review-check.yml@ref
+    with:
+      trigger-labels: '["CI", "preview", "skip-tests"]'
+      milestone-name: 'Validating'
+    secrets:
+      BOT_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
+```
+
```

Comment:
I think it could help providing also a short snippet of the workflow with the milestone event trigger